src/Dependencies/PooledObjects/ArrayBuilder.cs

```diff
@@ -270,9 +271,21 @@ public ImmutableArray<U> ToDowncastedImmutable<U>()
         /// <summary>
         /// Realizes the array and disposes the builder in one operation.
         /// </summary>
+        [PerformanceSensitive(
```

Comment:
This attribute can't be applied here. The code in PooledObjects is part of source nuget package and is references in other repos that do not have this attribute.